content/sources/repositories.adoc

```diff
@@ -2,37 +2,36 @@
 
 == Platform codebase
 
-=== Platform 5
+=== Platform 6
 
 eXo Platform is built from several projects stored in different {link-github-exoplatform}[*Git repositories*].
-You will find below the list of repositories used to build *eXo Platform 5*.
+You will find below the list of repositories used to build *eXo Platform 6*.
 
 [[plf_components]]
-.Overview of eXo Components for PLF 5.0
-image::plf5-components.png[Platform Components, align=center, link={imagesdir}/plf5-components.png]
+.Overview of eXo Components for PLF 6.0
+image::plf6-components.png[Platform Components, align=center, link={imagesdir}/plf6-components.png]
```

Comment:
We should keep the diagrams of PLF 5 also and explain how the architecture has evolved IMO. This guide is also for maintainers who will still work on PLF 5.

Reply:
done